tests/jballerina-semtype-port-test/src/test/resources/test-src/type-rel/mutable-record-t.bal

```diff
@@ -0,0 +1,82 @@
+type I record {|
+    int x;
+|};
+
+type S record {|
+    string x;
+|};
+
+type IS record {|
+    int|string x;
+|};
+
+// @type IorS < IS
```

Comment:
Here we are using the `@type` prefix but in other tests (like `mutable-record1.bal`) we are not using it. Not sure which I prefer (since unlike nBallerina I presume we are not going to have other prefixes such as `@output` and `@panic`) but I think we need to enforce one pattern, otherwise we'll accidentally use the wrong one and I assume test will ignore it

Reply:
This was done considering the maintainability aspects. We can directly copy all the nBallerina-type tests bal files without any modifications and run them in the jBallerina. I prefer maintaining the same file names and the same content here as well for the sake of maintainability.

Reply:
Added verification enforcing `\\  @type T1 REL T2` format with [5a1cc7b](https://github.com/ballerina-platform/ballerina-lang/pull/42362/commits/5a1cc7b153c114b00c8be0fffca7aa7420d9172e) as per our offline discussion.